packages/maria/src/lib/QueryHandler.ts

```diff
@@ -21,18 +26,52 @@ export class QueryHandler<StoredValue = unknown> {
     }
   }
 
+  public async deleteMetadata(key: string): Promise<void> {
```

Comment:
Would it be more efficient to store as a key:value database rather than one key containing all the metadata, seems inefficient

Reply:
There is a metadata table and a row for each separate provider instance.